packages/patrol_cli/lib/src/commands/test.dart

```diff
@@ -179,12 +180,31 @@ See https://github.com/leancodepl/patrol/issues/1316 to learn more.
       );
     }
 
+    final dartDefineFromFilePaths = stringsArg('dart-define-from-file');
```

Comment:
L183-200 are pretty much the same as L125-142 in `build_android.dart` and L192-209 in `develop.dart`.

I think it's the same behavior and we could extract it and have in one place, WDYT?

Reply:
in the end you just want some function that returns `mergedDartDefines` and `dartDefineFromFilePaths`

Reply:
I've extracted this common code to new function